Src/WitsmlExplorer.Frontend/components/ContentViews/CurveValuesView.tsx

```diff
@@ -219,13 +219,13 @@ export const CurveValuesView = (): React.ReactElement => {
       case DownloadOptions.All:
         exportAll();
         break;
-      case DownloadOptions.IntervalOfData:
-        exportSelectedIndexRange();
+      case DownloadOptions.SelectedRange:
+        exportSelectedRange();
         break;
       case DownloadOptions.SelectedIndexValues:
         exportSelectedDataPoints();
     }
-    downloadOptions = DownloadOptions.IntervalOfData;
+    downloadOptions = DownloadOptions.SelectedRange;
   };
 
   const exportSelectedIndexRange = useCallback(() => {
```

Comment:
Remove this old function (only used in a dependency list, remove there also)

Reply:
fixed